spec/user-stories.md

```diff
@@ -1,5 +1,25 @@
 ## Functional Stories
 
+### <dfn>Generic Storage</dfn>
+
+Bob wants to store and organize his data in a `[=storage=]` available online, so that data could be accessed from any device, any location, at any time. Bob wants to be free to store any type of resource in the `[=storage=]`. Bob wants his `[=storage=]` to allow for resources to be added, updated and/or deleted.
+
+### <dfn>Portable Storage</dfn>
+
+Bob wants the ability and choice to host the `[=storage=]` himself, or delegate the hosting to a `[=storage provider=]`. The `[=storage provider=]` may impose certain limitations, such as of type or size, and Bob may accept that if informed in advance. Bob also wants the ability to move (aka port) his `[=storage=]` to another `[=storage provider=]` as desired, without loosing any access to existing data or metadata, so that Bob is not dependent in perpetuity on one `[=storage provider=]`.
+
+Porting a `[=storage=]` from one provider to another should work the same even when Bob hosts the `[=storage=]` himself. In that situation Bob acts as `[=storage provider=]` as well. 
+
+### <dfn>Sharing Access</dfn>
+
+As an author with authoring permission, Guinan wants to use the sharing interface of her authoring tool to manage access permissions to her article, so that Deanna can review, Seven-of-Nine can co-edit, and Torres can view the progress. All `[=user=]`(s), Guinan, Deanna, Seven-of-Nine and Torres can identify themselves within the authoring tool using their personal online identities.
```

Comment:
*the authoring tool* might suggest that everyone uses the same tool, is that intended?

As for *identify themselves within the authoring tool*, is this detail important for this use case or maybe that the author can use those online identities when they manage access permissions.